src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/VisitRepository.kt

```diff
@@ -100,4 +102,20 @@ interface VisitRepository : JpaRepository<Visit, Long>, JpaSpecificationExecutor
     "SELECT CASE WHEN (COUNT(v) = 1) THEN TRUE ELSE FALSE END  FROM Visit v WHERE v.reference = :bookingReference AND v.visitStatus = 'BOOKED' "
   )
   fun isValidBookingReference(bookingReference: String): Boolean
+
+  @Transactional
+  @Modifying
+  @Query(
+    "Update visit SET  modify_timestamp = :modifyTimestamp WHERE id=:id",
+    nativeQuery = true
```

Comment:
We could change this from nativeQuery to a JPA query.

Reply:
native could be much faster!